.travis.yml

```diff
@@ -33,5 +34,5 @@ script:
 after_script:
 - cat .kitchen/logs/kitchen.log
 before_install:
-- openssl aes-256-cbc -K $encrypted_deb391b403a8_key -iv $encrypted_deb391b403a8_iv
-  -in travis-ci.pem.enc -out travis-ci.pem -d
+- 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then openssl aes-256-cbc -K $encrypted_deb391b403a8_key -iv $encrypted_deb391b403a8_iv
```

Comment:
See https://docs.travis-ci.com/user/pull-requests#Pull-Requests-and-Security-Restrictions